spec/lib/sequent/generator/project_spec.rb

```diff
@@ -58,7 +58,9 @@
     expect(File.read('blog-with_special-symbols/Rakefile')).to include("require './blog_with_special_symbols'")
   end
 
-  it 'has working example with specs' do
+  # Ignore for now until we find a way to correctly test the template project
```

Comment:
I think this is fixed by @werdlerk in master

Reply:
It breaks on my setup, some confusion with nested bundlers/`Gemfile`s, etc. I've not been able to fix this.

Also, the setup on master tests against the latest version published on rubygems, not the version that you are currently developing.